### PR TITLE
Correct translation string formatting

### PR DESCRIPTION
**Pull request by afestein**
Thanks for proposing a pull request.

To help us review the request, please complete the following:
- [x] sign contributor license agreement: https://developers.facebook.com/opensource/cla
- [x] submit against our `:dev` branch, not `master`.
- [x] describe the change (for example, what happens before the change, and after the change)

There were a couple of errors in the formatting of the `lrpoj` translation strings which were causing issues when compiling an app which uses the SDK.

Prior to this change, the app compile would fail with a JSON error `Property List error: Unexpected character / at line 1`. With these changes the app successfully compiles.

### CODE REVIEW

File: FacebookSDKStrings.bundle/Resources/hr.lproj/FacebookSDK.strings [hunk 1]
```diff
@@ -26,7 +26,7 @@
 "LikeButton.Like" = "Sviđa mi se";
 
 /* The label for the FBSDKLikeButton when the object is currently liked. */
-"LikeButton.Liked" = "Označeno sa "sviđa mi se"";
+"LikeButton.Liked" = 'Označeno sa "sviđa mi se"';
```
Comment (felixclack) [inline FacebookSDKStrings.bundle/Resources/hr.lproj/FacebookSDK.strings]: I don't think this is a valid string for the `strings` format. It doesn't allow for the use of single quotes.

Comment (felixclack) [inline FacebookSDKStrings.bundle/Resources/hr.lproj/FacebookSDK.strings]: I think it should be `"LikeButton.Liked" = "Označeno sa \"sviđa mi se\""

**Comment by dzhuowen** [comment]
Thanks for the PR!

File: FacebookSDKStrings.bundle/Resources/hr.lproj/FacebookSDK.strings (hunk 1 above)
Comment (afestein) [inline FacebookSDKStrings.bundle/Resources/hr.lproj/FacebookSDK.strings]: Hmm... already :shipit: 😟

**Pull request merged**